invenio_github/models.py

```diff
@@ -190,38 +177,31 @@ def get(cls, user_id, github_id=None, name=None, check_owner=True):
         :raises: :py:exc:`~sqlalchemy.orm.exc.MultipleResultsFound`: if
                  multiple repositories with the specified GitHub id and/or name
                  exist.
-        :raises: :py:exc:`RepositoryAccessError`: if the user is not the owner
-                 of the repository.
         """
-        repo = cls.query.filter((Repository.github_id == github_id) |
-                                (Repository.name == name)).one()
-        if (check_owner and repo and repo.user_id and
-                repo.user_id != int(user_id)):
-            raise RepositoryAccessError(
-                user=user_id, repo=name, repo_id=github_id)
+        # TODO check how many cases we have where github_id returns multiple results
+        repo = cls.query.filter(
+            (Repository.github_id == github_id) | (Repository.name == name)
+        ).one()
```

Comment:
what about `one_or_none` so higher level classes dont need to deal with SQL (data layer) exceptions like `NoResultsFound`?